style.css

```diff
@@ -571,18 +583,604 @@ footer h6 {
+    height: 26.76px;
+    width: 40%;
+    margin: 0 20%;
+    font-size: 14px;
+    font-weight: 400;
+    text-align: center;
+    position: relative;
+    left: 40px;
+    align-self: center;
+  }
+
+  #section-main button {
+    content: '';
+    width: 244px;
+    height: 70px;
+    padding: 1em;
+    float: left;
+    margin-right: 0;
+    margin-top: -30px;
+    font-size: 24px;
+    margin-left: 2%;
+  }
+
+  #btn-imgA {
+    display: inline-block;
+    position: absolute;
+    left: 42%;
+    margin-top: -20px;
+  }
+
+  #second-section-bg-img {
+    display: none;
+  }
+
+  #section-main {
+    width: 100%;
+    margin: 0;
+  }
+
+  #section-main::after {
+    content: url(./images/image-geometry1.png);
+    margin-right: 0;
+    position: absolute;
+    top: 105px;
+    right: 6.3%;
+    z-index: 10;
+    width: 40%;
+    height: 1600px;
+  }
+
+  #project-sections {
+    width: 100%;
+    margin-right: 0%;
+    margin-left: 0%;
+  }
+
+  .contact-form h2 {
+    height: 110px;
+    width: 40%;
+    font-family: 'Playfair Display', sans-serif;
+    margin: 0 30%;
+    font-size: 34px;
+    letter-spacing: 0.2em;
+    font-weight: 400;
+    text-align: center;
+  }
+
+  #project-sections h2 {
+    margin-left: 0;
+    transform: rotate(-90deg);
+    font-size: 110px;
+    position: absolute;
+    top: 1200px;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+  }
+
+  #full-section-div1 {
+    content: '';
+    float: right;
+    margin-top: 0;
+    margin-right: 0;
+    width: 60%;
+    height: 200px;
+    position: absolute;
+    right: 0;
+    margin-bottom: 0;
+  }
+
+  #imj-poject-section2 {
+    width: 30%;
+  }
+
+  #list-of-project2-img {
+    display: block;
+    position: absolute;
+    width: 465px;
+    height: 270px;
+    left: 1110px;
+    top: 1745px;
+  }
+
+  #list-of-project2-imgb {
+    display: block;
+    width: 350.75px;
+    height: 175.42px;
+    position: absolute;
+    top: 2012.5px;
+    right: 0%;
+  }
+
+  #imj-project-section img {
+    display: none;
+  }
+
+  #list-of-project1 {
+    float: right;
+    margin-top: 0;
+    margin-right: 0;
+    width: 60%;
+    height: 353px;
+    cursor: pointer;
+    position: absolute;
+    right: 0;
+    top: 1218px;
+    margin-bottom: 50px;
+  }
+
+  #list-of-project2 {
+    width: 27%;
+    margin-right: 0%;
+    cursor: pointer;
+    position: absolute;
+    right: 0;
+    top: 1570px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+  }
+
+  #list-of-project2::before {
+    content: url(./images/image-geometry2.png);
+    position: absolute;
+    top: -18px;
+    left: -124.8%;
+  }
+
+  .list-of-ptoject1 {
+    display: none;
+  }
+
+  #full-section-div2 {
+    display: block;
+    float: left;
+    width: 40%;
+    height: auto;
+    margin-left: 1%;
+    margin-top: 550px;
+    margin-right: 20%;
+  }
+
+  #full-section-div3 {
+    display: block;
+    width: 63.6%;
+    height: auto;
+    margin-left: 0;
+    margin-top: 910px;
+  }
```

Comment:
- [ ] Both of these regular projects should be aligned in the same row as it is in the Figma template below. The project on the left should take approximately 25% of the screen. The project on the right should take approximately 25% of the screen too and the image below it should be attached to it.
- [ ] The project on the right is being overlapped with some "background-image" in some screen sizes. Kindly fix it too.
- [ ] Fix the responsiveness of the tags inside the project on the right.
Your Project:
![image](https://user-images.githubusercontent.com/61919617/172937205-8b6e3510-dd64-4f78-bbd8-0a5218cd5379.png)
Figma Template:
![image](https://user-images.githubusercontent.com/61919617/172937264-fd11f19c-b3a3-4d98-b8f3-d7a0ae8bac9c.png)